dbt-cta/actblue/models/sources.yml

```diff
@@ -66,9 +66,6 @@ sources:
         tests:
           - dbt_expectations.expect_column_values_to_be_of_type:
               column_type: timestamp
-      - name: _airbyte_data
-        tests:
-          - not_null
   - name: paid_contributions_stream
     columns:
       - name: _airbyte_ab_id
```

Comment:
ope, @Frydafly this is not your fault because you inherited this 🙃  but can we change all the tests for `_airbyte_ab_id` and make them instead test uniqueness on the `_hashid` fields for each table?